### DISCUSSION TITLE
Issue in supended of account when clone a new repo

### DISCUSSION THREAD

**Discussion by muhammadmuzammil-dev** [community/community#120872] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

##General

##Account Suspension

My account was suspended for no valid reason also I didn't received any suspension email for account "0xaira". So I have created a new account to continue my open-source. Now when I am trying to push code to my new account it's showing :
remote: Your account is suspended. Please visit https://support.github.com/ for more information.
unable to access 'https://github.com/airajena/portfolio.git/': The requested URL returned error: 403

This is really pathetic experience by GitHub support. They are not fixing my account suspension issues . I have raised ticket. Also you are not allowing to create any account from my laptop. Maybe you have blacklisted my IP address. I request you to please fix this. I am already depressed seeing my previous account ban. It's really hurting. I can't take this anymore

git clone https://github.com/muhammadmuzammil-dev/myproject.git
Cloning into 'myproject'...
remote: Your account is suspended. Please visit https://support.github.com for more information.
fatal: unable to access 'https://github.com/muhammadmuzammil-dev/myproject.git/': The requested URL returned error: 403

**Comment by ebndev** [comment]
Hi @muhammadmuzammil-dev,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion and any duplicate discussions.

We appreciate your understanding!
